Exercise_2.java

```diff
@@ -1,3 +1,7 @@
+// Time Complexity :O(nlogn)
+// Space Complexity :O(n)
+// Did this code successfully run on Leetcode :yes

```

Comment:
The worst case Time Complexity for QuickSort is O(n^2) and the Worst Case Space Complexity is O(logn)